accounting/invoice/templates/without_bank_account/without_bank_account.html

```diff
@@ -0,0 +1,125 @@
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -
+-->
+
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="{{ site.domain }}">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="{% static 'vendor/css/bootstrap.min.css' %}"/>
+    <link rel="stylesheet" type="text/css" href="{% static 'without_bank_account/css/without_bank_account.css' %}"/>
+  </head>
+  <body>
+    <div class="container">
+      <div class="invoice">
+        <header>
+          <div class="row">
+            <div class="col-xs-6">
+              <h1><strong>{{ invoice.provider }}</strong></h1>
+              {% if invoice.provider.occupation %}
+                <h4>{{ invoice.provider.occupation }}</h4>
+              {% endif %}
+            </div>
+            <div id="invoice-metadata" class="col-xs-6">
+              <span id="invoice-number">Invoice Number: {{ invoice.number }}</span><br>
+              <span id="invoice-created">Created: {{ invoice.date|date:"F d, Y" }}</span><br>
+              <span id="invoice-billing-dates">Billed for {{ invoice.billing_start_date|date:"F d, Y" }} to {{ invoice.billing_end_date|date:"F d, Y" }}</span>
+              <span id="invoice-due">Due: {{ invoice.due_date|date:"F d, Y" }}</span>
+            </div>
+          </div>
+          <div class="row">
+            {% for user_type, user in users %}
+              <div class="col-xs-6">
+                <div id="{{ user_type }}-details">
+                  <h5 class="user-header"><strong>{{ user_type|upper }}</strong></h5>
+                  <span class="user-name">{{ user }}</span><br>
+                  <span class="user-address-line-1">{{ user.address.address_line1 }}</span><br>
+                  {% if user.address.address_line2 %}
+                    <span class="user-address-line-2">{{ user.address.address_line2 }}</span><br>
+                  {% endif %}
+                  <span class="user-city">{{ user.address.city }}</span>,
+                  {% if user.address.state %}
+                    <span class="user-state">{{ user.address.state }}</span>,
+                  {% endif %}
+                  <span class="user-zipcode">{{ user.address.zipcode }}</span><br>
+                  <span class="user-country">{{ user.address.country.name }}</span>
+                  {% if user.vat %}
+                    <br><span class="user-vat">VAT number: {{ user.vat }}</span>
+                  {% endif %}
+                </div>
+              </div>
+            {% endfor %}
+          </div>
+          {% if invoice.extra_text %}
+            <div class="row">
+              <div id="extra" class="col-xs-12">
```

Comment:
@UmanShahzad I noticed that the default template uses `col-xs-6` here -- any specific reason for using different classes?

Reply:
In Bootstrap we use `*-6` for when the div should be taking up 6 of the available 12 columns in the layout. Since this is the only div in the row, we give it all 12.

Reply:
Good thing to note here, maybe I should mention it somewhere at some point: WebKit HTML-to-PDF (wkhtmltopdf) does note support `media` queries, so that's why I'm using `xs` here *plus* and older version of Bootstrap for it. Any other size (`md` for example), or Bootstrap4, use media queries and thus wkhtmltopdf cannot handle them.